templates/archive.html

```diff
@@ -6,6 +6,7 @@
 
 {# Set locale for date #}
 {% set date_locale = macros_translate::translate(key="date_locale", default="en_GB", language_strings=language_strings) %}
+{% set short_date_format = macros_translate::translate(key="short_date_format", default="en_GB", language_strings=language_strings) %}
```

Comment:
🛠️ `default` here should be `"%d %b"`, no?

Reply:
I mean to establish a standard method, with long_date_format and short_date_format defined in i18n, it can display the right format as specified. For example, in China, the practice is to place the month before the date.